docs/source/data.rst

```diff
@@ -3,36 +3,145 @@ Data
 =========================
 
 
-Access to the raw disdrometer measurements and metadata is available through a
-separate GitHub repository : `disdrodb-data <https://github.com/ltelab/disdrodb-data>`__.
+Users can make their own data accessible to the community. Disdrodb
+provides a central storage for code (readers), issues and metadata.
+However, the raw data itself must be stored by the data provider due to
+size limitations.
 
-The `disdrodb-data` GitHub repository includes:
+Two types of data must be distinguished:
 
-* The stations measurements in the form of a JSON file with URLs links to the downloadable data (not included in the repository due to GitHub's file size limitation).
-* The stations metadata in YAML file format.
-* The stations issue in YAML file format.
-* Some python code to download the datasets using the URLs provided in the JSON files.
-* Some python code to curate the metadata archive.
+-  Station Raw Data:
 
+   -  Stores disdrometer measurements for days, weeks, and years.
+   -  This dataset can be very heavy.
+   -  No central storage is provided.
+
+-  Station Metadata and Issues:
+
+   -  Stores metadata on disdrometer measurements or measurement issues.
+   -  This dataset should be light.
+   -  Central storage is provided in the ``disdro-data`` Git repository.
+   -  The metadata folder contains a YAML metadata file called
+      ``metadata.yml``. It has a ``data_url`` key that references the
+      raw data. Only one zip file per station is allowed.
+
+How to download metadatas and issues ?
+--------------------------------------
 
 You can clone the disdrodb-data repository with
 
-.. code-block:: bash
+.. code:: bash
 
    git clone https://github.com/ltelab/disdrodb-data.git
 
+However, if you plan to add new data or metadata to the archive, first
+fork the repository on your GitHub account and then clone the forked
+repository.
+
+How to upload metadatas and issues?
+------------------------------------
+
+Do you want to contribute to the project with your own data? Great! Just
+follow these steps:
+
+1. Fork the ``disdro-data`` Git repository.
+
+2. Create a new branch.
+
+   .. code:: bash
+
+      git checkout -b "reader-<data_source>-<campaign_name>"
+
+3. Add your data source, campaign names, and station name to the current
+   folder structure.
+
+4. Add your metadata YAML file for each station ``station_name.yml``, in
+   a metadata directory in the campaign directory. We recommend you
+   copy-paste an existing metadata YAML file to get the correct
+   structure.
+
+5. (Optional) Add your issues YAML files, for each station
+   ``station_name.yml``, in an issue directory located in the campaign
+   directory. We recommend you copy-paste an existing metadata YAML file
+   to get the correct structure.
+
+6. Test that the integration of your new dataset functions by deleting
```

Comment:
Not clear here. You mean after that the PR has been merged, or after that you push on the personal fork?
Here I would add a BIG WARNING: to first git add, commit and push code on the repo ... to avoid loosing the work done !

Reply:
This should go after point 7. no?